src/covering.ml

```diff
@@ -1265,6 +1377,14 @@ let rec covering_aux env evars data prev clauses path (ctx,pats,ctx' as prob) le
 			   mkRel var) sortinv
 		in args, !argref
 	      in
+              (* Don't forget section variables. *)
+              let secvars =
+                let named_context = Environ.named_context env in
+                  List.map (fun decl ->
+                    let id = Context.Named.Declaration.get_id decl in
+                    Constr.mkVar id) named_context
+              in
+              let secvars = Array.of_list secvars in
```

Comment:
There's a named_context_instance function no? We can't use it because of let bindings maybe?